Dockerfiles/agent/Dockerfile

```diff
@@ -120,3 +120,15 @@ HEALTHCHECK --interval=2m --timeout=5s --retries=2 \
 VOLUME ["/var/run/s6", "/etc/datadog-agent", "/var/log/datadog", "/tmp"]
 
 CMD ["/init"]
+
+
+################################################################
+#  Sanity checks on the image contents                         #
+#  Build the release artifact with "--target release" to skip  #
+################################################################
+
+FROM release AS testing
+ARG WITH_JMX
+
+COPY test_*.py /
+RUN python /test_image_contents.py -v
```

Comment:
does that mean we'll be shipping the image with this extract file?

Reply:
No, because we are adding [`--target release`](https://docs.docker.com/engine/reference/commandline/build/#specifying-target-build-stage---target) to the`BUILD_ARG` [that are passed to docker build](https://github.com/DataDog/datadog-agent/pull/2994/files#diff-96edf7a6f008de9e928d04e1ae5e12a5R813)
This makes the second `docker build` stop at [the `release` stage](https://github.com/DataDog/datadog-agent/pull/2994/files#diff-c0e62e43e3f3ef321994872f8fef948dR50) and skip this testing stage.